recipes/cmake/3.x.x/conanfile.py

```diff
@@ -38,28 +39,30 @@ def requirements(self):
             self.requires("openssl/1.1.1s")
 
     def validate(self):
-        if self.settings.os == "Macos" and self.settings.arch == "x86":
+        if self.info.settings.os == "Macos" and self.info.settings.arch == "x86":
```

Comment:
https://github.com/conan-io/conan/issues/12671#issuecomment-1341034465 its been a while but I think we should be able to look at closing this

```suggestion
        if self.settings.os == "Macos" and self.settings.arch == "x86":
```

Reply:
Hi @prince-chrismc,
Sorry, I was thinking I needed to wait for the updated version of Conan 2 to be deployed but I guess that shouldn't matter since the CCI CI checks are not building with 2.0 yet. Just pushed this change